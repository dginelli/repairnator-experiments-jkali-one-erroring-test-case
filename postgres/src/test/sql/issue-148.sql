CREATE TABLE issue148 (null_field text[]);
INSERT INTO issue148 VALUES (ARRAY['one', null, 'two']);
CREATE INDEX idxissue148 ON issue148 USING zombodb (zdb('issue148', ctid), zdb(issue148)) WITH (url='http://localhost:9200/');

SELECT * FROM issue148 WHERE zdb('issue148', ctid) ==> 'null_field:null';
DROP TABLE issue148;