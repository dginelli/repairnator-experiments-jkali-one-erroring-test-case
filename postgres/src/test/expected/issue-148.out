CREATE TABLE issue148 (null_field text[]);
INSERT INTO issue148 VALUES (ARRAY['one', null, 'two']);
CREATE INDEX idxissue148 ON issue148 USING zombodb (zdb('issue148', ctid), zdb(issue148)) WITH (url='http://localhost:9200/');
SELECT * FROM issue148 WHERE zdb('issue148', ctid) ==> 'null_field:null';
 null_field 
------------
(0 rows)

DROP TABLE issue148;
